include/algorithms/weak_learner/weak_learner_training_batch.h

```diff
@@ -57,10 +57,10 @@ namespace interface1
  *      - \ref interface1::Model "Model" class
  *      - Result class
  */
-class DAAL_EXPORT Batch : public classifier::training::Batch
+class DAAL_EXPORT Batch : public classifier::training::interface1::Batch
```

Comment:
Does it mean that Batch class is deprecated now as there is no interface2::Batch?

Reply:
weak_learner namespace is fully deprecated